.github/workflows/macos.yml

```diff
@@ -0,0 +1,59 @@
+name: MacOS CI
+
+on: [push]
+
+jobs:
+  build-gcc8:
+    name: Build with GNU GCC 8
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Build kitty
```

Comment:
Did you mean "Build mockturtle" here? (same below)

Reply:
Thanks, great catch!